ruff.toml

```diff
@@ -126,21 +133,3 @@ ban-relative-imports = "all"
 
 [flake8-type-checking]
 strict = true
-
-[per-file-ignores]
-"docs/**/*.py" = ["INP001", "E402"]
-"python/**/__init__.py" = ["F401"]
-"python/pecos/circuits/qasm/std_gates.py" = ["F401"]
-"python/pecos/engines/hybrid_engine_multiprocessing.py" = ["BLE001"]
-"python/pecos/reps/pypmir/types.py" = ["F401"]
-"tests/**/*.py" = ["INP001", "ANN", "FBT001"] # ["D100", "D103", "B018", "FBT001"]
-# TODO: comment to work on better typing
```

Comment:
I'll add back in these per directory ignores as I am currently using these to slowly add in annotations by turning ANN back on and concentrating on un-ignored directories.